docs/_sources/usage/default_values.md.txt

```diff
@@ -0,0 +1,65 @@
+```{toctree}
+:maxdepth: 2
+```
+
+# Default values
+
+Testing SQL queries can often involve repetitive setup for mock tables. In SQLMock, one effective way to streamline this process is by using default values. By setting reasonable defaults, you can significantly reduce the boilerplate code in your tests, especially when dealing with multiple input tables or complex queries. Let’s explore how you can efficiently implement this.
+
+## Utilizing Default Values in MockTable Fields
+
+Defining default values at the field level in your mock tables is straightforward. The default argument in the field definition allows you to set default values, which are particularly useful for ensuring that joins and other query functionalities operate correctly.
+
+Here's an example:
+
+```python 
+@table_meta(table_ref="data.users")
+class UserTable(BigQueryMockTable):
+    user_id = col.Int(default=1)
+    user_name = col.String(default="Mr. T")
+
+# Create instances of the UserTable with various combinations of defaults and specified values
+users = UserTable.from_dicts([
+    {}, # Uses default values --> {"user_id": 1, "user_name": "Mr. T"}
+    {"user_id": 2}, # Overrides user_id but uses default for user_name
+    {"user_id": 3, "user_name": "Nala"} # No defaults used here
+])
+```
+
+## Setting Mock Defaults with table_meta
+
+
+When defining your MockTable classes, the `table_meta` decorator accepts a `default_inputs` argument.
+The Mock instances passed here, will be used as defaults in the `from_mocks` method.
+
+Consider this example:
+
+```python 
+@table_meta(
+    query_path="./examples/test_query.sql",
+    default_inputs=[UserTable([]), SubscriptionTable([])] # We can provide defaults for the class if needed.
+)
+class MultipleSubscriptionUsersTable(BigQueryMockTable):
+    user_id = col.Int(default=1)
+
+# Setting up different scenarios to demonstrate the use of defaults
+users = UserTable.from_dicts([{"user_id": 1}, {"user_id": 2}])
```

Comment:
**nitpicking: 🪡** consider using the same formatting for from_dicts as with _subscriptions_ below (as in break the lines down into one-per-user)